src/xrpld/shamap/detail/SHAMapInnerNode.cpp

```diff
@@ -44,6 +45,17 @@ SHAMapInnerNode::SHAMapInnerNode(
 
 SHAMapInnerNode::~SHAMapInnerNode() = default;
 
+void
+SHAMapInnerNode::partialDestructor()
+{
+    intr_ptr::SharedPtr<SHAMapTreeNode>* children;
+    // structured bindings can't be captured in c++ 17; use tie instead
```

Comment:
nit: I suppose we can use it now?

Reply:
Updated here + 3 other places to use modern structured bindings.